src/renderer/components/catalog/GridTagButton.tsx

```diff
@@ -19,10 +21,10 @@ interface IBaseProps {
 // ReturnType<typeof mapStateToProps>
 // ReturnType<typeof mapDispatchToProps>
 // tslint:disable-next-line: no-empty-interface
-interface IProps extends IBaseProps, RouteComponentProps {
+interface IProps extends IBaseProps, ReturnType<typeof mapStateToProps> {
 }
 
-class GridTagButton extends React.Component<IProps, undefined> {
+class GridTagButton extends React.Component<IProps> {
```

Comment:
Can we please continue to use the exact same consistent notation for `extends React.Component<PROPS, STATE>`, which is to use `{}` for unspecified PROPS (not `any`), and `undefined` for unspecified STATE (not `null` or empty).

Reply:
if you want, but it's not really important, it is `{}` by default

Reply:
of course not `any`or `null`

Reply:
It is just for code style consistency, search and replace, etc.